### PR TITLE
chore(releasing): Prepare v0.26.0 release

### PR DESCRIPTION
**Pull request by jszwedko**
TODO:

- [x] Ensure that all estimated JSON size work has been merged in
- [x] Determine if we need size estimation for metrics too
- [x] Cherry-pick issues from v0.26.0 milestone and add them to changelog
- [x] Confirm a user-facing description for #14889
- [x] Write description

### CODE REVIEW

File: website/cue/reference/releases/0.26.0.cue [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+
+				It can be set to `auto` to attempt to automatically determine the Elasticsearch
+				version by querying the Elasticsearch version endpoint.
+				"""
+			pr_numbers: [14918, 15082]
+			contributors: ["ktff"]
+		},
+		{
+			type: "feat"
+			scopes: ["sinks: elasticsearch"]
```
Comment (spencergilbert) [inline website/cue/reference/releases/0.26.0.cue:162]: 👀

**Review by spencergilbert — COMMENTED** [review]
Maybe I'm just still short on coffee, but I'm not seeing the commits that added support for the ES `api_version` option?

File: website/cue/reference/releases/0.26.0.cue [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+			breaking: true
+			description: """
+				The deprecated `swimlane` alias for the `route` transform was removed.
+
+				See [the upgrade
+				guide](/highlights/2022-11-07-0-26-0-upgrade-guide#lanes-parameter-route-transform-removal)
+				for more details.
+				"""
+			pr_numbers: [14903]
+		},
```
Comment (neuronull) [inline website/cue/reference/releases/0.26.0.cue:97]: This looks like a duplicate from lines 60-71

File: website/cue/reference/releases/0.26.0.cue [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+				Previously invalid field paths would be silently ignored at runtime.
+				"""
+			pr_numbers: [14887]
+		},
+		{
+			type: "fix"
+			scopes: ["file"]
+			description: """
+				The `file` source now requires `include` to be specified at compile time rather than
+				failing at runtime.
```
Comment (neuronull) [inline website/cue/reference/releases/0.26.0.cue:140]:
```suggestion
				The `file` source now fails to start if `include` option is not specified. The option was always required
				by the source, but no warnings or errors were being emitted if the option was not specified.
```

**Comment by jszwedko** [comment]
> Maybe I'm just still short on coffee, but I'm not seeing the commits that added support for the ES `api_version` option?

In the commit log at the bottom you mean? They are in there. Search for 0719f3439f9605bfeccc761ccf61fce7cc6c3b67.

**Review by spencergilbert — APPROVED** [review]

**Pull request merged**